recipes/fit-sne/bld.bat

```diff
@@ -0,0 +1,4 @@
+move "%RECIPE_DIR%\FItSNE.exe" "%LIBRARY_BIN%"
```

Comment:
You need `if errorlevel 1 exit 1` below each command

Reply:
Thanks, I updated the code.